target/linux/brcm63xx/dts/nb604n.dts

```diff
@@ -0,0 +1,124 @@
+/dts-v1/;
+
+#include "bcm6328.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Netcomm NB604N";
+	compatible = "brcm,nb604n", "brcm,bcm6328";
+
+	chosen {
+		bootargs = "rootfstype=squashfs,jffs2 noinitrd console=ttyS0,115200";
+		stdout-path = "serial0:115200n8";
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		rfkill {
+			label = "rfkill";
+			gpios = <&pinctrl 15 1>;
+			linux,code = <KEY_WLAN>;
+			debounce-interval = <60>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&pinctrl 23 1>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&pinctrl 24 1>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		usb {
+			label = "NB604N::usb";
+			gpios = <&pinctrl 1 1>;
+		};
+		inet_red {
+			label = "NB604N::red:internet";
+			gpios = <&pinctrl 2 1>;
+		};
+		dsl {
+			label = "NB604N::dsl";
+			gpios = <&pinctrl 3 1>;
+		};
+		power {
+			label = "NB604N::power";
+			gpios = <&pinctrl 4 1>;
+			default-state = "on";
+		};
+		power_red {
+			label = "NB604N::red:power";
+			gpios = <&pinctrl 5 1>;
+		};
+		wps {
+			label = "NB604N::wps";
+			gpios = <&pinctrl 10 1>;
+		};
+		inet {
```

Comment:
Use "internet" and add the color for all LEDs that are not unique (like you did for power_red).